config/solr_configs/synonyms.txt

```diff
@@ -18,6 +18,10 @@ ccc => cccc1,cccc2
 a\=>a => b\=>b
 a\,a => b\,b
 fooaaa,baraaa,bazaaa
+#The OGP file also includes these mappings
+aaafoo => aaabar
+bbbfoo => bbbfoo bbbbar
+cccfoo => cccbar cccbaz
```

Comment:
pretty sure these are fake/for testing?

Reply:
Yes, and they seem to extend the mappings for testing that are at the very beginning of our own file.

Reply:
I guess no harm keeping them in for testing 🤷🏻